swap.conf

```diff
@@ -27,10 +27,11 @@ zswap_zpool=zbud          # zbud z3fold
 # https://www.kernel.org/doc/Documentation/blockdev/zram.txt
 
 zram_enabled=0
-zram_size=$(($RAM_SIZE/4))      # This is 1/4 of ram size by default.
+zram_size=$(($RAM_SIZE*1))      # This is ram size by default (for deflate, or 1/4 for lz*)
```

Comment:
Why change defaults?

Reply:
I prefer just add another option, like zram_per_stream=<True|False>
With no more changes in config.

Reply:
I may rollback them, but performance of multicore deflate is comparable to singlecode lz4. Median convertion rate between lz4 and deflate is close to 4, thats why i multiplied it. (RAM/4 lz4 take same amount of physical memory as RAM deflate)